Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -683,6 +683,23 @@ - (void)updateUser:(NSDictionary *)dataFields mergeNestedObjects:(BOOL)mergeNest
     }
 }
 
+// documented in IterableAPI.h
+- (void)updateEmail:(NSString *)newEmail onSuccess:(OnSuccessHandler)onSuccess onFailure:(OnFailureHandler)onFailure
+{
+    if (newEmail) {
```

Comment:
can you just mark `newEmail` as non nullable?

Reply:
Yes, but non-nullability is not enforced in the Objective-C compiler: it will issue a warning but still let the user send a nil.

Reply:
got it. if you want to leave this check in that's fine.... but we should probably add non nullable anyways?

Reply:
Yup

Reply:
Actually, the entire header file is already wrapped in NS_ASSUME_NONNULL_BEGIN / NS_ASSUME_NONNULL_END, so every argument is implicitly non-null unless specified otherwise.

Reply:
....oh right, i forgot i did that 😅